src/Uno.UI.Wasm/ts/Windows/Devices/Midi/MidiInPort.ts

```diff
@@ -0,0 +1,33 @@
+﻿namespace Windows.Devices.Midi {
+	export class MidiInPort {
+		private static dispatchMessage: (serializedMessage: string) => number;
+
+		private static instanceMap: Array;
+
+		public static startMessageListener(encodedDeviceId: string) {
+			if (window.DeviceMotionEvent) {
+				this.dispatchMessage = (<any>Module).mono_bind_static_method("[Uno] Windows.Devices.Midi.MidiInPort:DispatchMessage");
+			}
+
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			var deviceId = decodeURIComponent(encodedDeviceId);
+			var input = midi.inputs.get(deviceId);
+			input.addEventListener("onmidimessage", MidiInPort.messageReceived);
+		}
+
+		public static stopMessageListener(encodedDeviceId: string) {
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			var deviceId = decodeURIComponent(encodedDeviceId);
+			var input = midi.inputs.get(deviceId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'input' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)